style.css

```diff
@@ -33,3 +33,30 @@ a {
 	font-size: 25px;
 }
 
+.calendar-icon.active {
+	display: block;
+	text-align: center;
+	color: #a6a2c5;
+	text-transform: uppercase;
+	padding: 15px;
+	text-decoration: none;
+	font-size: 25px;
+}
+
+.calendar-icon {
+	display: none;
+}
+
+.circle-icon {
+	display: none;
+}
+
+.circle-icon.active {
```

Comment:
Instead of creating a class for each icon, you could add a common class to every icon for example `link`, and then reuse it. :)

Reply:
Based on this CSS I think you are not familiar with CSS specificity and selectors.

Here's an interactive tutorial (aka a CSS game) to help you learn them: https://flukeout.github.io/